src/PlatformEngines.jl

```diff
@@ -40,14 +40,13 @@ function find7z()
     error("7z binary not found")
 end
 
-function probe_platform_engines!(;verbose::Bool = false)
```

Comment:
https://juliahub.com/ui/Search?q=probe_platform_engines%21&type=symbols

Maybe let them break?

Reply:
Yeah, they can pretty easily code around the absence or presence of this function. It will put a cap on compatible Julia versions for old releases of those packages, which is mildly annoying, but newer versions of the package can still support old and new Julias. Otoh, we could just leave this 🤷🏻‍♂️.

Reply:
I merged and opened issue on the packages that allow issues. NeXLCore doesn't allow issues so they don't get one.